dts/bindings/mipi-dbi/mipi-dbi-controller.yaml

```diff
@@ -0,0 +1,20 @@
+# Copyright 2023 NXP
+# SPDX-License-Identifier: Apache-2.0
+
+# Common fields for MIPI-DBI controllers
+
+include: base.yaml
+
+bus: mipi-dbi
+
+properties:
+  clock-frequency:
```

Comment:
What is the purpose of this property? Controller's output frequency can also be configured based on `mipi-max-frequency` device's property, isn't it?

Reply:
Same idea as the `clock-frequency` property for the SPI controller binding- lets say that your SPI peripheral is clocked at 50MHz, you could encode this in the `clock-frequency` property, then return an error if a peripheral tries to operate above 50MHz using your driver's `transceive` function. Many drivers get their peripheral clock at runtime using the clock control API, but I believe this could also serve as an alternative to that call